### DISCUSSION TITLE
Integration of Laboratory  Equipment & X-ray Machine with EMR

### DISCUSSION THREAD

**Discussion by shinapc** [nextgenhealthcare/connect#4599] [Q&A]
I am new to mirth, I have a project at hand to integrate Laboratory Equipment's, and X-ray machine with our Electronic Medical Records( EMR ) Solution, the EMR can speak HL7 via http/https protocol. Can any one please advise my on to go about this,
I know we need to build channels for every Equipment. if any individual/company have an existing channels or can build new channels for the list of  equipment bellow. please send me a private email to info@globalcarehms.com.

List Of Equipment
1) Horiba Pentra MS CRP
2) Horiba Pentra XLX
3) Yumizen H500
4) BIOLIS 30i
5) FUS 1000 Urinalysis
6) Render  MA120
7) Biosynex Malaria Cellscheck
8) Mindray BS 280
9) Techno Medica STAX 6E
10) Techno Medica GASTAT 730
11)Malgumi 800 Immunology.

Thanks So much

**Comment by pacmano1** [comment]
Lab equipment usually uses  ASTM for the transport of data.   Once received by Mirth, it can be transformed for delivery via HL7 to an EHR.   ASTM support is a premium (paid) plugin from Nextgen that comes with the Gold License Pack.   ngconnect-sales@nextgen.com  

If you are new to Mirth, I'd strongly advise you/your client to engage a senior Mirth consultant to kick start your project.

**Reply by shinapc** [reply]
Thanks very much for your reply, i have requested for a quotation for Mirth Premium version from next-gent, if you know of any Senior Mirth consultant i can engage for this job, please link me up.

**Comment by shinapc** [comment]
Is their any next-gen sales rep here, I  requested for a quotation for Mirth connect Premium version as suggested by pacmano1 above, to this email ngconnect-sales@nextgen.com, it is almost 3days now no response.

Secondly is their trail version of this Mirth connect Premium that include ASTM. that i can use for a proof of concept.
Thanks

**Reply by pacmano1** [reply]
They should reach out shortly - I reached out to them.    I don't work for them.

**Reply by muhammadc** [reply]
Hey there @shinapc. So sorry for the delay here. My name is Muhammad Chebli, I'm the VP of Solutions for interop here at NextGen. Please shoot me a quick email with your contact information and I'll have someone reach out ASAP. My email is mchebli at nextgen.com.

**Reply by shinapc** [reply]
i have send you the email

**Reply by muhammadc** [reply]
@shinapc, i'm sorry, I still haven't received any email from you.
